src/main/resources/pdiresources/transformations/merge-patient.ktr

```diff
@@ -440,7 +440,7 @@
     <server>${ETL_DATABASE_HOST}</server>
     <type>MYSQL</type>
     <access>Native</access>
-    <database>${ETL_MERGE_DATABASE}</database>
+    <database>merge_db</database>
```

Comment:
i think this is better maintained at $ETL_MERGE_DATABASE, set it in kettle.properties externally at-least to your value than hard code it

Reply:
good observation,  I  will fix  that

Reply:
seems like u forgot this!

Reply:
done